src/components/AqiGauge/InvalidSensor.tsx

```diff
@@ -29,43 +29,30 @@ const InvalidSensor: ({purpleAirId}: InvalidSensorProps) => JSX.Element = ({
   // this state variable stores which form of "at" we should use
   const [atString, setAtString] = useState(t('invalid.atPlural'));
```

Comment:
okay I'm not sure if this is an issue, but strangely the type of `atString` is `() => never`

Reply:
something to do with translate, I just added `'' + t('invalid.atPlural')` and now it says it's a string. The functionality works so maybe it's a VScode thing?